packages/server/tests/acceptance/htsPrecompile_v1.spec.ts

```diff
@@ -103,47 +90,72 @@ describe('HTS Precompile V1 Acceptance Tests', async function () {
     const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
     const tx = await baseHTSContract.createFungibleTokenWithCustomFeesPublic(accounts[0].wallet.address, HTSTokenContractAddress, {
       value: ethers.BigNumber.from('20000000000000000000'),
-      gasLimit: 10_000_000
+      gasLimit: 1_000_000
     });
     const txReceipt = await tx.wait();
     const { tokenAddress } = txReceipt.events.filter(e => e.event === 'CreatedToken')[0].args;
 
     return tokenAddress;
   }
 
+  it('should deploy baseHTSContract', async function () {
```

Comment:
I guess what i meant was you could create the the token, instantiate the contract from the address and associate in one test for the 3 scenarios.

Basically 
- `should createHTSToken` & `should associate to a token` would merge
- `should createNftHTSToken` & `should associate to a nft` would merge
- `should createHTSTokenWithCustomFees` & `should associate to a token with custom fees` would merge

`should finsh setting up contracts` shouldn't really be a thing.
The first parts can go into the tests where the Address is used and the wallet creations would be in the beforeAll

Just trying to make things contained but also ensure that it doesn't matter in what order tests are run